addons/l10n_sa_edi/models/res_partner.py

```diff
@@ -21,15 +21,19 @@ class ResPartner(models.Model):
         ('OTH', 'Other ID')
     ], default="OTH", string="Identification Scheme", help="Additional Identification scheme for Seller/Buyer")
 
-    l10n_sa_additional_identification_number = fields.Char("Identification Number (SA)",
-                                                           help="Additional Identification Number for Seller/Buyer")
+    l10n_sa_edi_additional_identification_number = fields.Char("Identification Number (SA)", help="Additional Identification Number for Seller/Buyer")
+    is_sa_company = fields.Boolean(compute="_compute_is_sa_company")
+
+    def _compute_is_sa_company(self):
```

Comment:
Is this field really necessary?
Can't you use `fiscal_country_codes` instead?

Reply:
Yeah it's enough my bad don't know why i didn't check on the partner, i went for a compute since we cannot do stuff like company.country_code ahah